code/lib/test/package.json

```diff
@@ -47,11 +48,11 @@
     "@storybook/core-events": "workspace:*",
     "@storybook/instrumenter": "workspace:*",
     "@storybook/preview-api": "workspace:*",
-    "@testing-library/dom": "^9.3.4",
-    "@testing-library/jest-dom": "^6.4.2",
-    "@testing-library/user-event": "^14.5.2",
-    "@vitest/expect": "1.3.1",
-    "@vitest/spy": "^1.3.1",
+    "@testing-library/dom": "10.1.0",
```

Comment:
Isn't this theoretically a breaking change?

> BREAKING CHANGES
> Minimum supported Node.js version is 18.0
> New version of aria-query changes various roles. Check out the changed tests in https://github.com/testing-library/dom-testing-library/commit/2c570553d8f31b008451398152a9bd30bce362b3 to get an overview about what changed.
- source: https://github.com/testing-library/dom-testing-library/releases

Reply:
See:
https://github.com/storybookjs/storybook/pull/27229#issuecomment-2132205782